gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.14.5
+version=29.14.6
```

Comment:
Since there is behavior change, let's bump major version on this.

Reply:
Decided to bump minor.